source/mir/random/tinflex/internal/area.d

```diff
@@ -226,9 +226,11 @@ body
             }
         }
     }
-    import std.math : isInfinity, isNaN;
-    if (isInfinity(area) || isNaN(area))
+    import std.math : isNaN;
+    if (isNaN(area))
```

Comment:
see R code:

```r
area <- function(cT, a,b,y, from,to) {
  ## -----------------------------------------------------------------------
  ## Compute area underneath hat or squeeze in particular interval.
  ## ------------------------------------------------------------------------
  ##   cT       ... parameter for transformation
  ##   a, b, y  ... intercept, slope and anchor point of transformed hat or squeeze
  ##   from, to ... range of integration
  ## ------------------------------------------------------------------------
  ## Return: area;
  ##         0 in case of an error.
  ## ------------------------------------------------------------------------

  ## Compute area ...
  area <- do.area(cT,a,b,y,from,to)

  ## ... and check result.
  
  if (! is.finite(area))
    ## We return Inf in all cases where 'area' is not finite (e.g. NaN or NA)
    area <- Inf

  if (area < 0) {
    ## Area is strictly negative.
    ## Two possible reasons:
    ## 1. The correct value of 'area' is extremely small or 0.
    ##    The negative number might result from cancelation.
    ##    Then 'area' can be set to 0.
    ## 2. Due to sever round-off error, the result is numerical garbage.
    ##
    ## We observed (2) in our experiments. So we discard the result and
    ## return 'Inf' (which means that the interval must be split).

    area <- Inf
  }

  return (area)
}
```